### DISCUSSION TITLE
Dynamic routes not working in nextjs

### DISCUSSION THREAD

**Discussion by pavlewebdev** [community/community#102873] [Programming Help] [answered]
### Body

i followed up tutorial and seems like my app is not working at all

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by KowyXYZ** [comment, marked as answer]
Did you check everything? (install packages, switch to client if needed)etc...

**Reply by pavlewebdev** [reply]
client is the problem thanks
